test/release.test.js

```diff
@@ -138,14 +134,3 @@ it('shows warning only once', () => {
   browserslist('last 2 versions')
   expect(console.warn).toHaveBeenCalledTimes(1)
 })
-
```

Comment:
I think it will be very good to have tests for `--update-db`. It will help us to be sure about what we are doing.